src/analysis/modals/AnalysisModal.ts

```diff
@@ -201,12 +201,15 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
         currentTool,
         [runtimeToolLabels.Jupyter, renderComputeModal],
         [runtimeToolLabels.RStudio, renderComputeModal],
-        [appToolLabels.GALAXY, () => renderAppModal(GalaxyModalBase, appToolLabels.GALAXY)],
-        [appToolLabels.CROMWELL, () => renderAppModal(CromwellModalBase, appToolLabels.CROMWELL)]
```

Comment:
I think we should leave this here for the GCP workspaces even though this feature is not released.